packages/component-base/test/url-utils.test.js

```diff
@@ -50,6 +50,29 @@ describe('url-utils', () => {
       expect(matchPaths('https://vaadin.com/docs/components', 'components')).to.be.true;
     });
 
+    describe('matchNested option', () => {
+      it('should match the exact path by default', () => {
+        expect(matchPaths('/users', '/users')).to.be.true;
+        expect(matchPaths('/users/', '/users')).to.be.false;
+        expect(matchPaths('/users/john', '/users')).to.be.false;
+        expect(matchPaths('/usersessions', '/users')).to.be.false;
+      });
+
+      it('should match the exact path when matchNested is false', () => {
+        expect(matchPaths('/users', '/users', { matchNested: false })).to.be.true;
+        expect(matchPaths('/users/', '/users', { matchNested: false })).to.be.false;
+        expect(matchPaths('/users/john', '/users', { matchNested: false })).to.be.false;
+        expect(matchPaths('/usersessions', '/users', { matchNested: false })).to.be.false;
+      });
+
+      it('should match nested paths when matchNested is true', () => {
+        expect(matchPaths('/users', '/users', { matchNested: true })).to.be.true;
+        expect(matchPaths('/users/', '/users', { matchNested: true })).to.be.true;
+        expect(matchPaths('/users/john', '/users', { matchNested: true })).to.be.true;
+        expect(matchPaths('/usersessions', '/users', { matchNested: false })).to.be.false;
```

Comment:
Should this also use `matchNested: true` as the test title says?

Reply:
Definitely, thanks. Fixed.